src/server/check-answers/index.js

```diff
@@ -0,0 +1,136 @@
+import { sectionToSummary } from '../common/templates/macros/create-summary.js'
+import { QuestionPage } from '../common/model/page/question-page-model.js'
+import { ConfirmationAnswer } from '../common/model/answer/confirmation/confirmation.js'
+import { Page } from '../common/model/page/page-model.js'
+import { submitApplication } from '../common/connectors/case-management/case-management.js'
+import { statusCodes } from '../common/constants/status-codes.js'
+import { QuestionPageController } from '../common/controller/question-page-controller/question-page-controller.js'
+
+/**
+ * @import {NextPage} from '../common/helpers/next-page.js'
+ * @import {ConfirmationPayload} from '../common/model/answer/confirmation/confirmation.js'
+ */
+
+const checkAnswersUrlPath = '/tb/submit/check-answers'
+
+export class ConfirmationPage extends Page {
+  urlPath = `/submit/confirmation`
+}
+
+const headingAndTitle = 'Check your answers before sending your application'
+
+export class SubmitSummaryPage extends QuestionPage {
+  question =
+    'Before you submit your application, you must select one of the declaration check boxes.'
+
+  get title() {
+    return headingAndTitle
+  }
+
+  get heading() {
+    return headingAndTitle
+  }
+
+  sectionKey = 'submit'
+  questionKey = 'check-answers'
+
+  view = `check-answers/index`
+
+  Answer = ConfirmationAnswer
+
+  nextPage() {
+    return new ConfirmationPage()
+  }
+
+  viewProps(req) {
```

Comment:
We can likely make our life easier by moving these view props down into the controller.

I believe super.handlePost takes viewProp args into it.

Then this becomes just a regular question page.